src/tests/tests/background.js

```diff
@@ -119,4 +124,22 @@
     }
   });
 
+  QUnit.test("DNT checking obeys user setting", (assert) => {
+    const NUM_TESTS = 5;
+
+    let done = assert.async(NUM_TESTS);
+
+    assert.expect(NUM_TESTS);
+
+    badger.isCheckingDNTPolicyEnabled = () => false;
```

Comment:
Ah, never mind, I somehow modified my local settings messing with the tests. Yeah, would be great to avoid this issue entirely.